components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java

```diff
@@ -163,8 +163,12 @@ public void init(ConfigurationContextService configurationContextService) {
         executorService.schedule(new Runnable() {
             @Override
             public void run() {
-                log.info("Starting polling event adapters");
-                getEventReceiverManagementService().startPolling();
+                try {
+                    log.info("Starting polling event adapters");
+                    getEventReceiverManagementService().startPolling();
+                } catch (Exception e) {
```

Comment:
@tishan89 
Thank you for pointing out. The reason behind this case is that we do not mention throwing any specific exception here and also it is similar to the scenario in https://github.com/wso2/carbon-analytics-common/blob/e632c8fa3c34072779a1178e85531bebcbd3a1e6/components/event-receiver/org.wso2.carbon.event.input.adapter.core/src/main/java/org/wso2/carbon/event/input/adapter/core/internal/CarbonInputEventAdapterService.java#L189 which handles a general exception. So, IMHO, I believe it is okay to leave it as it is. Please let me know if the above approach is incorrect